.github/workflows/release.yml

```diff
@@ -185,13 +181,41 @@ jobs:
           path: |
             ./tmp/${{ env.artifact_name }}.tgz
 
+      - name: Generate Release Notes
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          new_commit_hash=$(git rev-parse HEAD)
+          echo "New commit hash: $new_commit_hash"
+
+          previous_tag=$(git tag --sort=-version:refname | grep -E "^v0\.[0-9]+\.[0-9]+" | grep -Fv "${{ env.iota_ref }}" | head -1)
```

Comment:
why only v0 ?

Reply:
For some reason we have a v1.0.0 tag already 🤷‍♀️  I can change it but we need to delete that first

Reply:
It was removed